flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/handlers/JarRunHandler.java

```diff
@@ -92,7 +93,17 @@ public CompletableFuture<JarRunResponseBody> handleRequest(
             @Nonnull final DispatcherGateway gateway)
             throws RestHandlerException {
 
-        final Configuration effectiveConfiguration = new Configuration(configuration);
+        Map<String, String> requestJobFlinkConfig =
+                HandlerRequestUtils.fromRequestBodyOrQueryParameter(
+                        (request.getRequestBody()).getFlinkConfiguration(),
+                        () -> null,
+                        null,
+                        this.log);
```

Comment:
```suggestion
        Map<String, String> requestJobFlinkConfig = request.getRequestBody().getFlinkConfiguration();
```
Since there's no query parameter it doesn't make sense to call this method.